libraries/DataFlash/DataFlash_File.cpp

```diff
@@ -206,14 +190,13 @@ void DataFlash_File::periodic_fullrate(const uint32_t now)
 
 uint16_t DataFlash_File::bufferspace_available()
 {
-    uint16_t _head;
-    return (BUF_SPACE(_writebuf)) - critical_message_reserved_space();
+    return _writebuf.space() - critical_message_reserved_space();
```

Comment:
I'm going to fix this in an additional commit in this PR, just to keep the separation of the exchange from bug fixing.